pkg/db/db.go

```diff
@@ -0,0 +1,126 @@
+//Package db provides operations on SQLite DB
+package db
+
+import (
+	"database/sql"
+	// blank import to register sqlite3 driver
+	_ "github.com/mattn/go-sqlite3"
+)
+
+//Database type is sql.DB
+type Database sql.DB
```

Comment:
Not sure why the cast matters to you so much - It was my suggestion to use it like this because I don't see the need for a complete new object if the same works as well